MerlinAU.sh

```diff
@@ -1214,6 +1216,36 @@ _GetLatestFWUpdateVersionFromRouter_()
    echo "$newVersionStr" ; return "$retCode"
 }
 
+##---------------------------------------##
+## Added by ExtremeFiretop [2024-Apr-18] ##
+##---------------------------------------##
+_GetFirmwareVariantFromRouter_()
+{
+   local retCode=0  newVersionStr
+
+   newVersionStr="$(nvram get webs_state_info | sed 's/_/./g')"
+   if [ $# -eq 0 ] || [ -z "$1" ]
+   then
+       newVersionStr="$(echo "$newVersionStr" | awk -F '-' '{print $1}')"
+   fi
+
```

Comment:
@ExtremeFiretop,

OK, now delete lines #**1227** thru #**1230** from the "`_GetFirmwareVariantFromRouter_`" function:

```
if [ $# -eq 0 ] || [ -z "$1" ]
then
       newVersionStr="$(echo "$newVersionStr" | awk -F '-' '{print $1}')"
fi
```

That code will remove the "`-gnutonX`" tag from the F/W version string, so the next part of the code looking for it will never find it.